cmd/root.go

```diff
@@ -16,13 +16,21 @@ var StartDate string
 var EndDate string
 var Username string
 var Token string
+var LastWeek bool
+var ThisWeek bool
 
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
 	Use:   "github-activity",
 	Short: "Get your Github activity",
 	Long:  `Get PRs, reviews, and issues created during a specific time interval.`,
 	RunE: func(cmd *cobra.Command, args []string) error {
+		if LastWeek == true {
+			StartDate, EndDate = utils.GetLastWeekDates()
```

Comment:
the intended default behavior is to pull last week's activity, so maybe instead of adding a flag we call this if both start and end dates are null?

Reply:
I want it explicit since I am also adding --today and --last-week

Reply:
Default can remain as is